index.js

```diff
@@ -148,3 +148,31 @@ for (let i = 0; i < menuLis.length; i += 1) {
     menu.classList.replace('menu', 'invisible');
   });
 }
+
+function isLower(str) {
+  return !(/[a-z]/.test(str) && !/[A-Z]/.test(str));
+}
+
+const form = document.querySelector('.form');
+const emailInput = document.querySelector('[type=email]');
+const errorInput = document.querySelector('.error');
+
+function error() {
+  if (emailInput.validity.valueMissing) {
+    errorInput.innerHTML = '* Please enter an email address';
+    errorInput.classList.replace('invisible', 'error');
+  } else if (emailInput.validity.typeMismatch) {
+    errorInput.innerHTML = '* Please Enter Valid email (include "@" and a domain name)';
+    errorInput.classList.replace('invisible', 'error');
+  } else if (isLower(emailInput)) {
+    errorInput.innerHTML = '* Your email address should be all lowercase letters';
+    errorInput.classList.replace('invisible', 'error');
+  }
+}
+
+form.addEventListener('submit', (e) => {
+  if (!emailInput.validity.valid || isLower(emailInput.value)) {
+    error();
```

Comment:
- [ ] As per milestone requirements, The error message fits the design from the chosen template in Figma (e.g., font, colours, images, tests, margins). Kindly try to follow the design pattern and change the font colour and background colour as they fit to design.
![333](https://user-images.githubusercontent.com/57562869/147233479-4fc60536-3e68-457f-83d6-6a5b0ce729a1.png)


https://github.com/BereketRetta/MicroversePortfolio/blob/869ace65a115cef5c69a3ad9abeb29800a41c1c9/mobile.css#L647